src/EditorFeatures/CSharpTest/QuickInfo/SyntacticQuickInfoSourceTests.cs

```diff
@@ -34,6 +34,87 @@ public void Brackets_1()
             ExpectedContent("if (true)\r\n{"));
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public void ScopeBrackets_0()
+        {
+            TestInMethod(@"
+            if (true)
+            {
+                {
+                }$$
+            }
+",
+            ExpectedContent("{"));
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public void ScopeBrackets_1()
+        {
+            TestInMethod(@"
+            if (true)
+            {
+                // some
+                // comment
+                {
+                }$$
+            }
+",
+            ExpectedContent(
+@"// some
+// comment
+{"));
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public void ScopeBrackets_2()
+        {
+            TestInMethod(@"
+            if (true)
+            {
+                /* comment */
+                {
+                }$$
+            }
+",
+            ExpectedContent(
+@"/* comment */
+{"));
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public void ScopeBrackets_3()
+        {
+            TestInMethod(@"
+            if (true)
+            {
+                {
+                    // some
+                    // comment
+                }$$
+            }
+",
+            ExpectedContent(
+@"{
+    // some
+    // comment"));
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        public void ScopeBrackets_4()
+        {
+            TestInMethod(@"
+            if (true)
+            {
+                {
+                    /* comment */
+                }$$
```

Comment:
And, if you could, one for leading trivia with blank lines in between (I know it will work, but just for the scenario coverage):
```C#
// This comment is basically a dissertation!

// Part 1: Stuff.

// Part 2: More stuff!
{
}
```